assets/wizards/site-design/views/main/style.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../../shared/scss/colors';
+@use '../../../../shared/scss/colors';
```

Comment:
I think it should be safe to remove this, since none of the colour variables from the plugin are used in this file.

Reply:
Done in fa8dc6699d7f3726ec14ef3057596338ce7d230d